src/Canvas.cc

```diff
@@ -88,9 +88,9 @@ NAN_METHOD(Canvas::New) {
 
   Backend* backend = NULL;
   if (info[0]->IsNumber()) {
-    int width = info[0]->Uint32Value(), height = 0;
+    int width = info[0]->Uint32Value(Nan::GetCurrentContext()).FromMaybe(0), height = 0;
```

Comment:
I think that we should use the `Nan::To<...>(...)` API instead, to better support multiple versions of V8 🤔 

```suggestion
    int width = Nan::To<uint32_t>(info[0]).FromMaybe(0), height = 0;
```

Reply:
Fixed.